src/components/v5/common/MemberSignatureList/MemberSignatureList.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
+
+import { SpinnerLoader } from '~shared/Preloaders';
+import MemberSignature from '../MemberSignature';
+import { MemberSignatureListProps } from './types';
+
+const displayName = 'v5.common.MemberSignatureList';
+
+const MemberSignatureList: FC<MemberSignatureListProps> = ({
+  items,
+  isLoading,
+  title,
+}) => (
+  <div>
+    <h3 className="text-1 mb-2">{title}</h3>
+    {isLoading && (
+      <div className="flex justify-center">
+        <SpinnerLoader appearance={{ size: 'medium' }} />
+      </div>
+    )}
+    {!isLoading && !!items?.length && (
```

Comment:
What will be shown if there will be no items?

Reply:
Added `Members list is empty` if there are no items